repo/packages/K/kafka/0/config.json

```diff
@@ -1,6 +1,35 @@
 {
-  "type": "object",
-  "properties": {},
-  "additionalProperties": false,
-  "required": []
+    "type": "object",
+    "properties": {
+        "image": {
+            "type": "string",
+            "description": "Scheduler image",
+            "default": "https://s3-us-west-2.amazonaws.com/kafka-mesos/kafka-mesos-0.9.0-beta.zip"
+        },
+        "master": {
+            "type": "string",
+            "description": "--master option",
+            "default": "zk://master.mesos:2181/mesos"
+        },
+        "zk": {
+            "type": "string",
+            "description": "--zk option",
+            "default": "master.mesos:2181"
+        },
+        "api": {
+            "type": "string",
+            "description": "--api option",
+            "default": "http://$HOST:$PORT0"
+        },
+        "storage": {
+            "type": "string",
+            "description": "--storage option",
+            "default": "zk:/kafka-mesos"
+        },
+        "other": {
```

Comment:
Sure. I've renamed other -> otherOptions. Now it should be self-explainable. Pushed changes.